client/src/components/layout/ReviewList.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+const ReviewList = ({reviews, id}) => {
+  const reviewListItems = reviews.map(review => {
+    return (
+      <li key={id}>
+        {review.reviewText}
+      </li>
```

Comment:
Even though this li is small, it should be made into a component. React documentation recommends to do so whenever you are making a list of react elements (in this case a list of lis) 

Youll want this in the long run anyway!